setup.py

```diff
@@ -2,18 +2,16 @@
 from os import path
 from io import open
 
-import versioneer
-
 here = path.abspath(path.dirname(__file__))
 
 # Get the long description from the README file
 with open(path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
 setup(name='pynvml',
-      version=versioneer.get_version(),
-      cmdclass=versioneer.get_cmdclass(),
+      version='12.0.0',
       python_requires='>=3.6',
+      install_requires = ['nvidia-ml-py<13.0'],
```

Comment:
Think we should have a lower bound. Also think we need to guard against any dev or RC versions that could come out

```suggestion
      install_requires = ['nvidia-ml-py>=12.0.0,<13.0.0a0'],
```